runtime/stdlib/test_emulatorbackend.go

```diff
@@ -726,7 +726,7 @@ func (t *testEmulatorBackendType) newMoveTimeFunction(
 			if !ok {
 				panic(errors.NewUnreachableError())
 			}
-			blockchain.MoveTime(int64(timeDelta.ToInt(invocation.LocationRange)))
+			blockchain.MoveTime(float64(timeDelta) / sema.Fix64Factor)
```

Comment:
Casting an `int64` to `float64` could lose precision in some cases. We could instead pass `int64` (`Fix64Value` as is) to `MoveTime`, and change the parameter's metric to accept nano-seconds (it currently accepts seconds). But might need to pad-right if we do so, since `Fix64Factor` is 10^8 and nano-seconds factor 10^9. But then there's a risk of padding-right could overflow/underflow. So IDK which one is better.
In reality, these extreme cases would not be even used, so maybe we don't need to worry much, and just documenting might be enough?
@turbolent wdyt?

Reply:
Agreed, we shouldn't use floating point here